lib/ansible/modules/packaging/language/pip.py

```diff
@@ -743,9 +765,15 @@ def main():
 
         changed = changed or venv_created
 
-        module.exit_json(changed=changed, cmd=cmd, name=name, version=version,
-                         state=state, requirements=requirements, virtualenv=env,
-                         stdout=out, stderr=err)
+        if constraints:
```

Comment:
There is no reason to make this conditional.  We don't do this for `requirements` don't do it here too.  Just use a single `module.exit_json`